### PR TITLE
Update to latest accessibility developer tools

### PR DESCRIPTION
**Pull request by RickSBrown**
The primary purpose of this update is to consume API changes in the accessibility developer tools library which made this update non-trivial.

### CODE REVIEW

**Comment by RickSBrown** [comment]
@alice PTAL

File: src/extension/ExtensionAuditRule.js [hunk 1]
```diff
@@ -20,15 +20,13 @@ goog.provide('axs.ExtensionAuditRule');
...
-    axs.AuditRule.call(this, spec);
-
-    /** @type {boolean} */
-    this.requiresConsoleAPI = !!spec['opt_requiresConsoleAPI'];
+axs.ExtensionAuditRule = function(auditRule) {
+    goog.object.extend(auditRule, this);
+    return auditRule;
 };
-goog.inherits(axs.ExtensionAuditRule, axs.AuditRule);
+//goog.inherits(axs.ExtensionAuditRule, axs.AuditRule);
```
Comment (alice) [inline src/extension/ExtensionAuditRule.js]: Lingering commented-out line

**Comment by alice** [comment]
:+1: once single nit addressed.

**Comment by RickSBrown** [comment]
@alice I think maybe you need to give me a higher level of access to the extension project for me to be able to merge this pull request?

**Comment by alice** [comment]
Ah yeah, oops. I'll pull it, then if you want to keep contributing to the extension (I'm assuming you probably don't) I'll add you to the project.

**Pull request merged**